sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -75,6 +75,19 @@ export interface BearerTokenAuthenticationPolicyOptions {
     scopes: string | string[];
 }
 
+// @public
+export interface BlobLike {
+    size?: number;
+    stream: ReadableStream | NodeJS.ReadableStream | (() => ReadableStream | NodeJS.ReadableStream);
+    type?: string;
+}
+
+// @public
+export interface BodyPart {
+    body: ReadableStream | NodeJS.ReadableStream | Uint8Array | BlobLike;
```

Comment:
The name property is only handled in `formDataPolicy` (which does take `FileLike`), since it gets encoded in the `Content-Disposition` header. I think that specifying it here could be misleading because it kind of implies that the file name will be used when it actually isn't.

Reply:
Makes sense now 👍